client/src/main.tsx

```diff
@@ -13,7 +13,7 @@ createRoot(document.getElementById("root")!).render(
       <ThemeProvider>
         <App />
       </ThemeProvider>
-      <ReactQueryDevtools initialIsOpen={false} />
+      {import.meta.env.DEV && <ReactQueryDevtools initialIsOpen={false} />}
```

Comment:
Can this env variable be separated to a named constant?
May be hard to read or modify this when migrating, etc.